source/kraken/realtime.cpp

```diff
@@ -46,6 +46,22 @@ namespace navitia {
 
 namespace nd = type::disruption;
 
+static bool base_vj_exists_the_same_day(const type::Data& data, const transit_realtime::TripUpdate& trip_update) {
+    const auto& mvj = *data.pt_data->get_or_create_meta_vehicle_journey(trip_update.trip().trip_id(),
+                                                                        data.pt_data->get_main_timezone());
+    if (mvj.get_base_vj().empty()) {
```

Comment:
You can probably simplify code below using https://github.com/CanalTP/navitia/blob/d68c8536cd654c7777dff02ec9b34694ed3b50d8/source/type/type.cpp#L296